src/components/LessonList/components/PaginatedLessonList/components/LessonListItem/index.js

```diff
@@ -23,8 +24,17 @@ export default ({instructor, lesson}) => {
           </Heading>
         </Link>
 
+        {stateIsRequested ? null :
+          <Link to={`/instructors/${instructor.slug}`} className="no-underline">
+            <div className='mb43 flex items-center pt1'>
+              <img src={instructor.avatar_url} alt={instructor.full_name} className='w2 h2 br-pill mr3'/>
```

Comment:
There is an `Avatar` component for this; example use: https://github.com/eggheadio/egghead-instructor-center/blob/master/src/screens/Instructors/screens/Instructor/components/InstructorInfoCard/index.js#L12-L16